src/OpenLoco/Map/Tile.h

```diff
@@ -115,8 +115,6 @@ namespace OpenLoco::Map
             return type() == TClass ? (TType*)this : nullptr;
         }
 
-        uint8_t baseZ() const { return _base_z; }
```

Comment:
This _private_ `TileElement` member function was shadowing a _public_ `TileElementBase` member function.